Formula/s/sshpass.rb

```diff
@@ -0,0 +1,28 @@
+class Sshpass < Formula
+  desc "Non-interactive ssh password auth"
+  homepage "https://sourceforge.net/projects/sshpass/"
+  url "https://master.dl.sourceforge.net/project/sshpass/sshpass/1.10/sshpass-1.10.tar.gz"
+  sha256 "ad1106c203cbb56185ca3bad8c6ccafca3b4064696194da879f81c8d7bdfeeda"
+  license "GPL-2.0-only"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats
+    <<~EOS
+      Sshpass is a tool for non-interactivly performing password authentication
+      with SSH's so called "interactive keyboard password authentication".
+      Most user should use SSH's more secure public key authentiaction instead.
+
+      See `man sshpass` for more information.
+    EOS
+  end
+
+  test do
+    system "#{bin}/sshpass", "-V"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Already fixed in the newest commit, please check it when it's convenient. :)